capabilities/lib/zarf.ts

```diff
@@ -29,20 +29,18 @@ export async function updateZarfManagedImageSecrets(
   authToken: string,
 ): Promise<void> {
   try {
-    const namespace = await K8s(kind.Namespace).Get();
+    const namespace = await K8s(kind.Namespace)
+      .WithLabel(managedByLabel, "zarf")
+      .Get();
+
     const namespaces = namespace.items;
 
     for (const ns of namespaces) {
       try {
         await K8s(kind.Secret)
           .InNamespace(ns.metadata!.name!)
-          .WithLabel(zarfManagedByLabel, "zarf")
           .Get(zarfImagePullSecret);
       } catch (err) {
-        // Continue checking the next namespace if this namespace doesn't have a "private-registry" secret
-        if (JSON.stringify(err).includes("404")) {
-          continue;
-        }
         throw new Error(JSON.stringify(err));
       }
```

Comment:
this try: catch is no longer needed as the below SSA w/ `force: true` takes care of it + we know we are in the correct w/ the above ns get change

Reply:
Updated. Great eye on not needing this try/catch block and get secret call after filtering on namespace labels. This is much more clean and easier to follow now